README.md

```diff
@@ -1926,6 +1926,11 @@ arXiv, 2020. [Project page](https://sites.google.com/view/video2events/home), [C
     - DHP19 - 	DAVIS Human Pose Estimation and Action Recognition
     - DDD20 - End-to-End Event Camera Driving Dataset
 
+### Stereo Matching
+- <a name="MGehrig21ral"></a>Gehrig, M., Aarents, W., Gehrig, D., Scaramuzza, D.,  
```

Comment:
It's OK to start a section on stereo matching --> Stereo Depth Estimation
I would also put here previous datasets on the same topic. Some that come to my mind are:
MVSEC (RAL 2018)
Zhou (ECCV 2018)
Zhou (TRO 2021) HKUST sequence
Could you please add these and others to the list? Thanks